eox_audit_model/decorators.py

```diff
@@ -67,30 +73,34 @@ def get(self, request, *args, **kwargs):
     def decorator(func):  # pylint: disable=missing-docstring
         @wraps(func)
         def wrapper(*args, **kwargs):  # pylint: disable=missing-docstring
+            nonlocal data_filter, hidden_fields
             request = args[1]
-            data = request.data if request.data else request.query_params
+            input_parameters = []
+            request_data = request.data if request.data else request.query_params
+            data_filter = [] if data_filter is None else data_filter
+            hidden_fields = [] if hidden_fields is None else hidden_fields
 
-            # TODO: This is a hotfix. Consider addding all the filtered data for all items
-            # in the list.
-            if isinstance(data, list):
-                audit_data = None
-            else:
+            request_data = [request_data] if not isinstance(request_data, list) else request_data
```

Comment:
```suggestion
            request_data = request_data if isinstance(request_data, list) else [request_data]
```
If you're using both branches (`if-else`) don't start with the negation.

Reply:
@MoisesGSalas Done!